### DISCUSSION TITLE
Type Script Error checking breaks down after doing builds in Android Studio

### DISCUSSION THREAD

**Discussion by wtfNoNamesLeft** [quasarframework/quasar#16993] [CLI - Capacitor / Cordova modes] [answered]
I'm having an issue with capacitor compilation. I have an application that's targeted at both SPA and Capacitor. I can happily go along doing quasar dev, quasar build, etc.

To build the hybrid app I'm using android studio to build the capacitor apps specifically to generate signed bundles using Android Studio frankly because it's easier than command line. This is after doing the quasar dev/build -m capacitor -T android compile.

The issue I'm facing is that after doing the build in android studio for both debug and release bundles I start getting errors related to type script in my project in VSCode. Bascially what's happening is the tsconfig.json file is being ignored.

I can delete the \src-capacitor\android\app\build directory and all of my type script "errors" are gone. tsconfig, specifically the "paths" and a couple other things just start working.

My questions are:
1. Has anyone faced anything similar?
2. Are there downsides to excluding the build folder in my tsconfig.json file?
3. Am I just freeing up memory space for the Node TS checker and if so what happens as my project grows and that no longer works?

**Comment by rstoenescu** [comment, marked as answer]
Hi,

You should add an `exclude` option to your `/tsconfig.json` file, starting with (and adding whatever else you feel necessary):

```
{
   //...
   "exclude": [
      "./dist",
      "./.quasar",
      "./node_modules",
      "./src-capacitor",
      "./src-cordova",
      "./quasar.config.*.temporary.compiled*"
    ]
}
```
